buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -5,15 +5,15 @@ object Versions {
     // Plugin
     const val androidPluginVersion = "7.0.1"
     const val ktlintGradleVersion = "10.0.0"
-    const val dokkaVersion = "1.4.20"
+    const val dokkaVersion = "1.5.30"
     const val gradleVersionsPluginVersion = "0.36.0"
     const val nexusStagingVersion = "0.22.0"
 
     // Android
     const val minSdkVersion = 19
     const val compileSdkVersion = 30
     const val targetSdkVersion = 30
-    const val ndkVersion = "21.4.7075529"
+    const val ndkVersion = "27.0.11718014"
```

Comment:
If your NDK is less than 26, you will also need to update CMake.
Therefore, I increased the NDK to the officially recommended 27.